src/index.html

```diff
@@ -7,8 +7,61 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img src="images/logo.png" alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="ul">
+          <li class="li">
+            <a class="a is-active">Apple</a>
+          </li>
```

Comment:
Need to fix classes everywhere
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a class="nav__link is-active">Apple</a>
          </li>
```